package.json

```diff
@@ -54,10 +54,10 @@
     "ember-keyboard": "^4.0.0",
     "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-modal-dialog": "^2.4.4",
+    "ember-modal-dialog": "^3.0.0-beta.2",
```

Comment:
Why moving to a `beta` version?

Reply:
:trollface: . I'll wait for the final tag

Reply:
It seems that they aren't going to drop a final tag any time soon. Will close this.